float32 basePosition
float32 baseVelocity
bool baseStalled

float32 shoulderPosition
float32 shoulderVelocity
bool shoulderStalled

float32 elbowPosition
float32 elbowVelocity
bool elbowStalled
